designer/server/src/main/scala/pl/touk/nussknacker/ui/services/ScenarioAttachmentService.scala

```diff
@@ -0,0 +1,61 @@
+package pl.touk.nussknacker.ui.services
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.commons.io.IOUtils
+import org.apache.commons.io.input.BoundedInputStream
+import pl.touk.nussknacker.engine.api.process.{ProcessId, VersionId}
+import pl.touk.nussknacker.ui.config.AttachmentsConfig
+import pl.touk.nussknacker.ui.process.repository.ProcessActivityRepository
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+import pl.touk.nussknacker.ui.services.ScenarioAttachmentService.{AttachmentDataWithName, AttachmentToAdd}
+
+import java.io.InputStream
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Using
+
+class ScenarioAttachmentService(config: AttachmentsConfig, scenarioActivityRepository: ProcessActivityRepository)
```

Comment:
at the moment we have:
* http services defined in `pl.touk.nussknacker.ui.services`
* non-http services defined in `pl.touk.nussknacker.ui.services`
* non-http services scattered in the packages (e.g AllDeployedScenarioService`

Do you have any idea how to sort it out?

Reply:
I moved it to scenario (process) package. There are similar, non-http services which are connected with scenario (process).